README.md

```diff
@@ -47,22 +47,124 @@ classic-mceliece-rust = { version = "1.0", features = ["mceliece6960119"] }
 
 The `simple` example illustrates the API:
 ```rust
-use classic_mceliece_rust::{crypto_kem_dec, crypto_kem_enc, crypto_kem_keypair};
+use classic_mceliece_rust::{keypair, encapsulate, decapsulate};
+# #[cfg(feature = "alloc")]
```

Comment:
How do these look in the Github Markdown renderer?

Reply:
I have not tried that yet.. I mostly focused on making it nice in rustdoc.. Yeah, as you can see here, they don't look very nice: https://github.com/mullvad/classic-mceliece-rust/tree/rustcrypto_kem_api

And since the ones about Windows does not even solve the problem we should probably go another route here.